a: 0.5
b: 0.5
m: 5
mb: 0
g: 9.8
J: 0.4167
init_state: [-0.1901, 0.1902, 1.1029, 0.8114]

# terrain info
slopeAngle: [1.5,0,0,0,5,1,-3,-1,1.5,0,0,0,0,1,-3,-1]
slopeLength: [0.6,0.4,0.6,0.8,0.7,0.9,0.6,0.8,0.5,0.6,0.6,0.4,0.7,0.8,0.4,0.6]
slopeHeight: [0,0.01,0.01,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0]
